### PR TITLE
Update Arch install guide

### PR DESCRIPTION
**Pull request by NoaHimesaka1873**

### CODE REVIEW

File: docs/distributions/arch/installation.md [hunk 1]
```diff
@@ -35,22 +35,32 @@ You will need:
...
-9. Now in your `chroot`, follow the [Audio Config Guide](https://wiki.t2linux.org/guides/audio-config/).
+         1. Run `t2strap /mnt base linux-firmware iwd grub efibootmgr` (omit the `grub efibootmgr` packages from this if you intend to use systemd-boot as your bootloader). You can choose to use Xanmod kernel instead. In this case, append `-k xanmod` to `t2strap` command.
+
+8. Continue following the Arch Wiki's guide until you get to installing a bootloader.
 
-10. Add `apple-bce` to the `MODULES` list in `/etc/mkinitcpio.conf`, and then run `mkinitcpio -P`
+9. Add `apple-bce` to the `MODULES` list in `/etc/mkinitcpio.conf`, and then run `mkinitcpio -P`
 
-11. Add Redecorating's repository to `/etc/pacman.conf`, by adding this:
+10. Add repositories to `/etc/pacman.conf`, by adding this: (You can skip this if you used `t2strap`)
```
Comment (Redecorating) [inline docs/distributions/arch/installation.md:52]: could this go to step 2 under the pacstrap section of step 7?

File: docs/distributions/arch/installation.md [hunk 2]
```diff
@@ -35,22 +35,32 @@ You will need:
...
 6. Continue following the Arch Wiki's guide until "Install essential packages".
 
-7. Install the required packages into your new system with: `pacstrap /mnt base linux-t2 apple-bcm-wifi-firmware linux-firmware iwd grub efibootmgr` (omit the `grub efibootmgr` packages from this if you intend to use systemd-boot as your bootloader).
+7. Install the required packages into your new system.
 
-8. Continue following the Arch Wiki's guide until you get to installing a bootloader.
+    -    Using pacstrap (more vanilla Arch experience)
+
+         1. Run `pacstrap /mnt base linux-t2 apple-t2-audio-config apple-bcm-firmware linux-firmware iwd grub efibootmgr` (omit the `grub efibootmgr` packages from this if you intend to use systemd-boot as your bootloader). You can choose to use Xanmod kernel instead. In this case, replace `linux-t2` with `linux-xanmod-edge-t2`.
+
```
Comment (Redecorating) [inline docs/distributions/arch/installation.md:43]: (instructions for adding the repos could go here)

**Review by Redecorating — APPROVED** [review]
other than possibly moving the add repos step, lgtm

**Comment by NoaHimesaka1873** [comment]
@Redecorating done.

**Review by Redecorating — APPROVED** [review]

**Pull request merged**